src/VerifyWindowsHelloController.js

```diff
@@ -54,6 +59,17 @@ function (Okta, FormController, FormType, webauthn, FooterSignout, WindowsHelloE
                 clientData: assertion.clientData,
                 signatureData: assertion.signature
               });
+            })
+            .fail(function (error) {
+              switch (error.message) {
+              case 'AbortError':
+              case 'NotFoundError':
+              case 'NotSupportedError':
+                model.trigger('abort', error.message);
+                return transaction;
```

Comment:
I am wondering what happens here since we're returning a different transaction (the original transaction, not the current state which is the new factorTransaction). Are we still able to continue?